public/assets/Fork-Awesome-1.1.7/.github/ISSUE_TEMPLATE/Icon_request.md

```diff
@@ -0,0 +1,29 @@
+---
```

Comment:
maybe I'm wrong, but shouldn't other projects, that are being used, be pulled in dynamically by whatever builds the website rather than being checked into version control?